### ISSUE TITLE
While Building Stellar-Core in Ubuntu using Windows Subsystem for Linux

### ISSUE THREAD

**Issue by SabarishT** [stellar/stellar-core#2025]
# Issue type
   #006a4f9

# Version
 Stellar Core 10.3.0

# Environment and setup
   Ubuntu 18.04.1 LTS

# Issue description
 When I am running **make check** in terminal getting test failed at one point. Please find below screenshot for your reference what exactly i am getting. Please anyone help me regarding this issue. 

`/home/stellar-core/src/test/selftest-parallel: line 69: parallel: command not found
FAIL: test/selftest-pg
==================
1 of 1 test failed
==================
Makefile:2130: recipe for target 'check-TESTS' failed
make[3]: *** [check-TESTS] Error 1
make[3]: Leaving directory '/home/stellar-core/src'
Makefile:2253: recipe for target 'check-am' failed
make[2]: *** [check-am] Error 2
make[2]: Leaving directory '/home/stellar-core/src'
Makefile:2255: recipe for target 'check' failed
make[1]: *** [check] Error 2
make[1]: Leaving directory '/home/stellar-core/src'
Makefile:569: recipe for target 'check-recursive' failed
make: *** [check-recursive] Error 1`

![Capture3](https://user-images.githubusercontent.com/29983995/55003007-8482de00-4ffd-11e9-8c5e-0121cc2d2417.PNG)

**Comment by andersonmmi** [comment]
Ran into same issue just now on OS X.

**Comment by zealord** [comment]
Looks like the parallel package is missing when running this line (which run the test in parallel mode):

`cat $TESTS | parallel -j $nparts -n $BATCHSIZE --line-buffer --tag-string '{%}' --progress --halt-on-error 2 runpart '{%}' "{}"
`
It could be fixed on OS X via homebrew: `(sudo) brew install parallel`
For more: https://www.gnu.org/software/gsrc/

**Comment by SabarishT** [comment]
Thank you @zealord for the reply. After installation parallel package I am getting following failed status

![Capture4](https://user-images.githubusercontent.com/29983995/55074922-9759e900-50b7-11e9-9638-84606f1567b0.PNG)

**Comment by SabarishT** [comment]
Please anybody help me regarding above issue.

**Comment by vogel** [comment]
Provided information is not enough to give any advice. Please try doing a clean install, and if that does not help, attach full compilation log here (not screenshots of it, but full log - from beginning to error). That way we can see what is happening.

**Comment by MonsieurNicolas** [comment]
@SabarishT open a different issue with exact repro steps if you're still having problems

**Comment by Singtolife** [comment]
I also encountered a similar problem. However, the system did not give an obvious error message. I tried many methods without success. I don't know what to do.
![make check error1](https://user-images.githubusercontent.com/89182024/130735503-2d9de93b-fd9c-49cb-8642-22963ac75816.png)
![make check error2](https://user-images.githubusercontent.com/89182024/130735529-830fb2b3-a482-42e2-bdb5-4276a434c9ed.png)
